packages/config-schema/src/schema/netlify-toml.schema.json

```
@@ -0,0 +1,646 @@
+        "required": ["for"],
+        "additionalProperties": false
+      },
+      "x-taplo": {
+        "docs": {
+          "main": "Additional request headers to send in [proxy redirects](https://docs.netlify.com/routing/redirects/rewrites-proxies/#custom-headers-in-proxy-redirects)."
+        },
+        "links": {
+          "key": "https://docs.netlify.com/routing/redirects/rewrites-proxies/#custom-headers-in-proxy-redirects"
+        }
+      }
+    },
+    "functions": {
+      "title": "Functions",
+      "description": "Although there are default settings for Netlify Functions to help you get started, you can use this section for optional, custom configuration.",
+      "type": "object",
+      "additionalProperties": {
+        "description": "Options that only apply for this filter by name, including using glob patterns. If a function matches several configuration blocks containing one of these properties, the values are concatenated.",
+        "type": "object",
+        "properties": {
+          "node_bundler": { "$ref": "#/definitions/FunctionConfigNodeBundler" },
+          "external_node_modules": { "$ref": "#/definitions/FunctionConfigExternalNodeModules" },
+          "included_files": { "$ref": "#/definitions/FunctionConfigIncludedFiles" }
+        },
+        "additionalProperties": false
+      },
+      "properties": {
+        "deno_import_map": {
+          "title": "Import maps",
+          "description": "Relative path to an import map file to map module URLs to names.",
+          "type": "string"
+        },
+        "external_node_modules": { "$ref": "#/definitions/FunctionConfigExternalNodeModules" },
+        "included_files": { "$ref": "#/definitions/FunctionConfigIncludedFiles" },
+        "node_bundler": { "$ref": "#/definitions/FunctionConfigNodeBundler" },
+        "directory": {
+          "title": "Directory",
+          "description": "Custom path to your functions. The default location is `YOUR_BASE_DIRECTORY/netlify/functions`.",
+          "type": "string",
+          "default": "netlify/functions"
+        }
+      },
+      "x-taplo": {
+        "links": {
+          "key": "https://docs.netlify.com/configure-builds/file-based-configuration/#functions"
+        }
+      }
+    },
+    "edge_functions": {
+      "type": "array",
+      "default": [{ "path": "/path", "function": "function-name" }],
+      "items": {
+        "title": "Edge Functions",
+        "description": "Edge Functions connect the Netlify platform and workflow with an open runtime standard at the network edge. This enables you to build fast, personalized web experiences with an ecosystem of development tools.",
+        "type": "object",
+        "properties": {
+          "path": {
+            "type": "string",
+            "description": "The path pattern that needs to match for this edge function."
+          },
+          "function": {
+            "type": "string",
+            "description": "The name of the function."
```

Edit to be consistent

```suggestion
            "description": "Name of the function."
```